content/en/logs/explorer/visualize.md

```diff
@@ -43,15 +43,13 @@ The default **sort** for logs in the list visualization is by timestamp, with th
 
 **Note**: Sorting your table according to a specific field requires that you [declare a facet][1] beforehand.
 
-[1]: /logs/explorer/facets/
```

Comment:
It looks like removing these links is breaking the link in the page. 🧐

Reply:
oh weird, I did not do that, the link checker did... lemme see if I can fix it